version.properties

```diff
@@ -1,4 +1,4 @@
 # Version of the produced binaries.
 # The version is inferred by shipkit-auto-version Gradle plugin (https://github.com/shipkit/shipkit-auto-version)
-version=2.2.*
+version=3.0.*
```

Comment:
Can you rebase and read RELEASING.md? We won't be maintaining "version.properties" file any more ;)

Reply:
Done, thanks!